rust/actyx/ax/src/cmd/settings/mod.rs

```diff
@@ -25,6 +27,9 @@ pub enum SettingsOpts {
     Get(GetOpt),
     /// Get setting schemas from a node
     Schema(SchemaOpt),
+    /// Locally get/set/unset settings inside actyx-data directory
```

Comment:
```suggestion
    /// Locally get/set/unset settings inside ax-data directory
```

Reply:
currently supported operations are get/set/init; supporting `unset` would make sense from a consistency standpoint

Reply:
unset on the way